### PR TITLE
Allow use of access_token in github module

### PR DESCRIPTION
**Pull request by nclsHart**
Implements an alternative way to authenticate through GitHub API, see https://developer.github.com/v3/#authentication for more informations.

### CODE REVIEW

File: i3pystatus/github.py [hunk 1]
```diff
@@ -34,6 +35,7 @@ class Github(IntervalModule):
         ('unread_marker', 'sets the string that the "unread" formatter shows when there are pending notifications'),
         ("username", ""),
         ("password", ""),
+        ("access_token", ""),
```
Comment (enkore) [inline i3pystatus/github.py]: Can you put the link to the docs you mentioned here? Looks otherwise good, thanks.

**Comment by nclsHart** [comment]
@enkore done.

I also add a link to the tokens generation page.

**Comment by enkore** [comment]
Nice, thanks!

**Comment by ncoop** [comment]
Having an issue with this, but I don't think it has anything to do with your implementation, @nclsHart
```python
from json import loads
from subprocess import check_output
from requests import get


def report(data):
    if isinstance(data, list):
        print("list length: {}".format(len(data)))
    else:
        print(data)
    print()

full_url = "https://api.github.com/notifications" \
           "?access_token=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"

print("with curl:")
raw = check_output("curl --silent {}".format(full_url), shell=True)
data = loads(raw.decode("utf-8"))
report(data)

print("with requests:")
response = get(full_url)
data = loads(response.text)
report(data)
```

Result:
```
with curl:
list length: 3

with requests:
{'documentation_url': 'https://developer.github.com/v3', 'message': "Missing the 'notifications' scope."}
```

It's the same token, but GitHub won't take it if sent with `requests`?

**Comment by nclsHart** [comment]
I copy/paste your code and I get the following result: 

```
with curl:
list length: 1

with requests:
list length: 1
```

Maybe you should retry with another token.

**Comment by ncoop** [comment]
@nclsHart thanks for testing. It does work for me on other hosts.

**Pull request merged**